Resources/Prototypes/_Goobstation/MartialArts/carp.yml

```
@@ -0,0 +1,64 @@
+- type: martialArt
+  id: SleepingCarp
+  martialArtsForm: SleepingCarp
+  roundstartCombos: SleepingCarpMoves
+  randomDamageModifier: True
+  baseDamageModifier: 5
+  randomSayings:
+  - carp-saying-huah
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)